### PR TITLE
Add modal for the place pickers

### LINKED ISSUE #220: Add Modals for PlacePickers (FROM, TO)

### PR DESCRIPTION
**Pull request by FilipMessa**
close #220

### CODE REVIEW

File: apps/core/scenes/search/Placepickers.js [hunk 1]
```diff
@@ -9,39 +9,49 @@ import {
...
+        icon={<Icon name="airplane-takeoff" />}
+        value={props.travelFrom}
+      />
+      <TouchableWithoutFeedback onPress={props.handlePlaceSwitchPress}>
+        <View style={styles.placeSwitch}>
+          <Icon name="replace" color="#7F91A8" />
+        </View>
+      </TouchableWithoutFeedback>
+      <TripInput
+        onPress={createHandler(props.setModalType, MODAL_TYPE.DESTINATION)}
```
Comment (tbergq) [inline apps/core/scenes/search/Placepickers.js]: Why this approach over just creating a class and using the class method which will already have access to the `setModalType`

File: apps/core/scenes/search/SearchModal.js [hunk 2]
```diff
@@ -65,6 +70,20 @@ class SearchModal extends React.Component<Props> {
...
         )}
+        {[MODAL_TYPE.ORIGIN, MODAL_TYPE.DESTINATION].includes(
+          this.props.modalType,
+        ) && (
+          <View
+            style={{
+              margin: 25,
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}
```
Comment (tbergq) [inline apps/core/scenes/search/SearchModal.js]: Please create a style with `StyleSheet.create` instead

Comment (FilipMessa) [reply apps/core/scenes/search/SearchModal.js]: this is just temporary so I will change it in next PR

Comment (tbergq) [reply apps/core/scenes/search/SearchModal.js]: ok

File: apps/core/scenes/search/Placepickers.js (hunk 1 above)
Comment (FilipMessa) [reply apps/core/scenes/search/Placepickers.js]: You mean create a class from PlacePicker? 
And then inside create something like `this.handleFromClick` and `this.handleToClick`.
Do I understand right?

Comment (tbergq) [reply apps/core/scenes/search/Placepickers.js]: Yes

Comment (tbergq) [reply apps/core/scenes/search/Placepickers.js]:
```
class PlacePickers extends React.Component<Props> {
	onPress = () => {this.props.setModalType(type)}
	render() {
		return ()
	}
}
```

Comment (FilipMessa) [reply apps/core/scenes/search/Placepickers.js]: OK it is a good idea so I will change it

Comment (FilipMessa) [reply apps/core/scenes/search/Placepickers.js]: Done

**Pull request merged**